### ISSUE TITLE
[enhancement] PSX Multidisk Support

### ISSUE THREAD

**Issue by fluffymadness** [digitalLumberjack/recalbox-os#248]
As discussed in the recalbox forum:
http://blog.recalbox.com/forums/topic/playstation-memory-cards-and-save-states-for-multi-disk-games/

The problem is, if you have multiple isos e.g FF7-CD1.iso FF7-CD2.iso you have different memorycard files. One for each iso...you would have to rename the files or make symlinks...which is kinda inconvenient.

A good solution would be to use eboot files (.pbp). You can convert multiple isos to a single eboot file.
You will have the following benefits:

- Multiple isos in one file (no savegame rename problem)
- Compression

The only thing that has to be done to get this working would be to assign new standard special commands for recalbox:

- Eject Disk
- Next Disk
- Previous Disk

Then it is possible to switch the disks ingame without leaving the emulator :)
http://blog.petrockblock.com/forums/topic/best-way-for-psx-multidisc-games/

Would be nice if you could include this in the next build.
Thanks.

**Comment by digitalLumberjack** [comment]
Yes very good idea.

**Comment by rockaddicted** [comment]
Yes indeed, there is a discussion in the forum's french section about this.
http://blog.recalbox.com/forums/topic/roms-playstation-et-nombre-de-jeux/page/2/#post-5194

**Comment by Laddyacris** [comment]
Add to wiki for awhile 
https://github.com/digitalLumberjack/recalbox-os/wiki/Gestion-multidisc-PSX

**Comment by Guzilli** [comment]
Better Multidisc support would be great, so +1

**Comment by sanderaido** [comment]
I see it's possible now with eboot files but can I just change the rom on the fly? An example would be the Monster Rancher game, where you can create new breeds by placing in different CD's. I wouldn't want to make a huge eboot file just to try out some different disk images, is there an option to just change the image without reseting the emulator?

**Comment by gabrielcrowe** [comment]
I suppose this might also be fixed by making multiple files have the same memory card too. I'm not sure if even this would work, but it seems like the most elegant solution.

"Share memory card with..."

**Comment by gabrielcrowe** [comment]
I've just stumbled across this problem and the solution was complicated for the person who i built my system for. They just cant comprehend the menus and browsing for files. I've sort of necro'd this

**Comment by nwildner** [comment]
Not to necrobump, but a solution i've found was to configure a custom key to "Open Tray", "Disk index +1", "Disk index -1" that would be very useful for pbp/multidisk files.

- L3(left stick click) - Open Tray
- Left Stick, Left - Disk index -1
- Left Stick, Right - Disk index +1

It would be very good if something near this setup comes already configured on Recalbox.
